Source/SwiftLintBuiltInRules/Rules/Idiomatic/PreferKeyPathRule.swift

```diff
@@ -6,11 +6,21 @@ struct PreferKeyPathRule: Rule {
     var configuration = PreferKeyPathConfiguration()
 
     private static let extendedMode = ["restrict_to_standard_functions": false]
+    private static let ignoreIdentity = ["ignore_identity_closures": true]
+    private static let extendedModeAndIgnoreIdentity = [
+        "restrict_to_standard_functions": false,
+        "ignore_identity_closures": true,
+    ]
 
     static let description = RuleDescription(
         identifier: "prefer_key_path",
         name: "Prefer Key Path",
-        description: "Use a key path argument instead of a closure with property access",
+        description: """
```

Comment:
Let's keep the existing description as it's currently also used as the violation message and add a `rational` property instead to explain this detail.